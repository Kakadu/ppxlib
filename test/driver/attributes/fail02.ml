let f x = 1 [@@deprecatd "..."]
