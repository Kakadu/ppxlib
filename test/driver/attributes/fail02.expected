File "fail02.ml", line 1, characters 15-24:
Error: Attribute `deprecatd' was not used.
Hint: Did you mean deprecated?
